src/it/resources-noncompilable/com/google/checkstyle/test/chapter5naming/rule527localvariablenames/InputPatternVariableNameEnhancedInstanceofTestDefault.java

```diff
@@ -65,17 +70,17 @@ public void t(Object o1, Object o2) {
       System.out.println("done");
     }
 
-    {
-      while (!(o1 instanceof String _aa)) {
-        // violation above 'Pattern variable name '_aa' must match pattern'
-        L3:
-        break L3;
+      {
+        while (!(o1 instanceof String _aa)) {
+          // violation above 'Pattern variable name '_aa' must match pattern'
+          L3:
+          break L3;
+        }
+        while (o1 instanceof String aa_) {
+          // violation above 'Pattern variable name 'aa_' must match pattern'
+          aa_.length();
+        }
```

Comment:
known indentation defect in our config: https://github.com/checkstyle/checkstyle/issues/14294#issuecomment-1904385363

Reply:
Issue is moved into Todo to be stand out of crowd